frontend/src/components/PPViewer/PPViewerContainer.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import PPViewer from './PPViewer';
+
+interface IPPViewerContainer {
```

Comment:
I dont now, should we create types file for cases when we just adding interface for props?. In this case, we can create types and IPPViewerProps will extends from IPPViewerContainerProps

Reply:
Don't know either. @k-krasilnikova 's opinion required. But IMHO I don't think we should extract this somewhere outsde

Reply:
Let's create a separate folder inside PPViewer for types and move all the ty;es declarations inside

Reply:
@k-krasilnikova is this rule common to all folders?